src/main/java/eu/dissco/core/digitalspecimenprocessor/service/EqualityService.java

```diff
@@ -165,8 +164,7 @@ private static boolean isEqual(JsonNode currentJson, JsonNode json, String id) {
   private boolean entityRelationshipsAreEqual(EntityRelationship currentEntityRelationship,
       EntityRelationship entityRelationship) {
     if (currentEntityRelationship == null || entityRelationship == null) {
-      log.warn("Null ER!");
-      return currentEntityRelationship == null && entityRelationship == null;
+      return false;
```

Comment:
Should we not log something when both of them are null?

Reply:
I'll add the message for now